.github/workflows/examples-old-nodes.yml

```diff
@@ -5,6 +5,14 @@ on:
 
 jobs:
   e2e-old-nodes:
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_HOST_AUTH_METHOD: trust
```

Comment:
nice, i didn't know this one

Reply:
It's the same we use in the docker-compose that comes with a few examples ;)

<img width="514" alt="CleanShot 2023-02-24 at 19 44 12@2x" src="https://user-images.githubusercontent.com/51714798/221264702-5f30c4de-b072-4f0a-8eae-f1da40a2b464.png">